sirbot_pyslackers/endpoints/slack/messages.py

```diff
@@ -13,8 +13,15 @@
 from .utils import ADMIN_CHANNEL, HELP_FIELD_DESCRIPTIONS
 
 LOG = logging.getLogger(__name__)
-STOCK_REGEX = re.compile(r"\b(?P<asset_class>[cs])\$(?P<symbol>\^?[A-Z.]{1,5})\b")
+STOCK_REGEX = re.compile(
+    r"\b(?P<asset_class>[cs])\$(?P<symbol>\^?[A-Z.]{1,5})-?(?P<currency>[A-Z]{3})?\b"
+)
 TELL_REGEX = re.compile("tell (<(#|@)(?P<to_id>[A-Z0-9]*)(|.*)?>) (?P<msg>.*)")
+CRYPTO_CURRENCIES = {
```

Comment:
These aren't cryptos, perhaps a rename is appropriate? Quote currencies? Fiat currencies?

Reply:
Yes, a poor name. I kind of meant CURRENCIES_FOR_CRYPTO I guess, but FIAT_CURRENCIES is better. I'll change it.

Reply:
Renamed to FIAT_CURRENCY